cdk/kittyhawk/.projenrc.js

```diff
@@ -4,8 +4,8 @@ const common = require('../projen-common');
 const project = new TypeScriptProject({
```

Comment:
I think we'll have to run projen one more time before releasing to ensure all of our files are set up correctly

Reply:
Yeah B)